docs/handcraft.rst

```diff
@@ -0,0 +1,2049 @@
+       ...:         },
+       ...:     },
+       ...:     'divisible': False,
+       ...:     'refillable': False,
+       ...:     'updatable': False,
+       ...:     'id': asset_id,
+       ...: }
+
+.. note:: In a signature-less system, the values for the keys:
+    
+    * ``'divisible'``
+    * ``'refillable'``
+    * ``'updatable'``
+    * ``'id'``
+
+    could all be generated on the server side.
+
+metadata
+^^^^^^^^
+.. ipython::
+
+    In [0]: metadata = {
+       ...:     'data': {
+       ...:         'planet': 'earth',
+       ...:     },
+       ...:     'id': metadata_id,
+       ...: }
+
+.. note:: In a signature-less system, the value for the ``id`` could be
+    generated on the server side.
+
+operation
+^^^^^^^^^
+
+.. ipython::
+
+    In [0]: operation = 'CREATE'
+
+.. important::
+
+    Case sensitive, all letters must be capitalized.
+
+conditions
+^^^^^^^^^^
+The purpose of the condition is to lock the transaction, such that a valid
+fulfillment is required to unlock it. In the case of signature-based schemes,
+the lock is basically a public key, such that in order to unlock the
+transaction one needs to have the private key.
+
+Let's review the condition payload of the prepared transaction, to see what we
+are aiming for:
+
+.. ipython::
+
+    In [0]: prepared_creation_tx['transaction']['conditions'][0]
+
+The difficult parts are the condition details and uri. We''ll now see how to
+geenrate them using the ``cryptoconditions`` library.
+
+.. ipython::
+
+    In [0]: from cryptoconditions import Ed25519Fulfillment
+
+    In [0]: ed25519 = Ed25519Fulfillment(public_key=alice.verifying_key)
+
+generate the condition uri:
+
+.. ipython::
+
+    In [0]: ed25519.condition_uri
+
+So you have the condition uri now.
+
+As for the details:
+
+.. ipython::
+    
+    In [0]: ed25519.to_dict()
+
+We can now easily assemble the ``dict`` for the condition:
+
+.. ipython::
+
+    In [0]: condition = {
+       ...:     'amount': 1,
+       ...:     'cid': 0,
+       ...:     'condition': {
+       ...:         'details': ed25519.to_dict(),
+       ...:         'uri': ed25519.condition_uri,
+       ...:     },
+       ...:     'owners_after': (alice.verifying_key,),
+       ...: }
+
+Let's recap and set the ``conditions`` key:
+
+.. ipython::
+ 
+    In [0]: from cryptoconditions import Ed25519Fulfillment
+
+    In [0]: ed25519 = Ed25519Fulfillment(public_key=alice.verifying_key)
+
+    In [0]: condition = {
+       ...:     'amount': 1,
+       ...:     'cid': 0,
+       ...:     'condition': {
+       ...:         'details': ed25519.to_dict(),
+       ...:         'uri': ed25519.condition_uri,
+       ...:     },
+       ...:     'owners_after': (alice.verifying_key,),
+       ...: }
+
+    In [0]: conditions = (condition,)
+
+The key part is the condition uri:
+
+.. ipython::
+
+    In [0]: ed25519.condition_uri
+
+.. todo:: Explain the basics of the above. cc, 4, 20, the hash of the key, 96
```

Comment:
You can just point out that its part of the `cc` spec and point them to the ietf draft